flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/rules/logical/PushFilterIntoTableSourceScanRuleTest.java

```diff
@@ -86,6 +86,39 @@ public void setup() {
                         + ")";
```

Comment:
Better to add some ITCase To verify the correctness of the data after push down

Reply:
Sure. Could you please point me to where I can add ITcase for this fix?

Reply:
Added ITCases for the new feature.